ebos/ecloutputblackoilmodule.hh

```diff
@@ -50,8 +50,14 @@ BEGIN_PROPERTIES
 NEW_TYPE_TAG(EclOutputBlackOil);
 
 NEW_PROP_TAG(ForceDisableFluidInPlaceOutput);
+NEW_PROP_TAG(ForceDisableProdRepOutput);
```

Comment:
I have not looked at this PR as a whole - and I am not the one to fight in the review process - but still: Can we please avoid adding new tags?

Reply:
Ok, I will remove the new tags created. What would be the best way to give the option for the tables not be printed? Can I just leave it as always printing?

Reply:
Honestly I don't know; what I would do was just to introduce a hardcoded bool variable early in the call stack; then you are in shape to actually adjust that bool at a later stage.